### PR TITLE
fix: custom rpcs return bn not big int

### PR DESCRIPTION
**Pull request by gregdhill**
Signed-off-by: Gregory Hill <gregorydhill@outlook.com>

### CODE REVIEW

File: src/apis/vaults.ts [hunk 1]
```diff
@@ -176,12 +176,12 @@ export class DefaultVaultsAPI {
         try {
             const collateralization = newCollateral
                 ? await customAPIRPC.vaultRegistry.getCollateralizationFromVaultAndCollateral(
-                    vaultId,
-                    this.wrapCurrency(newCollateral),
-                    onlyIssued
-                )
+                      vaultId,
```
Comment (savudani8) [inline src/apis/vaults.ts:179]: watch out for the indent

File: src/apis/vaults.ts [hunk 2]
```diff
@@ -181,7 +181,7 @@ export class DefaultVaultsAPI {
                     onlyIssued
                 )
                 : await customAPIRPC.vaultRegistry.getCollateralizationFromVault(vaultId, onlyIssued);
-            return this.scaleUsingParachainGranularity(collateralization);
+            return this.scaleUsingParachainGranularity(new Big(collateralization));
```
Comment (savudani8) [inline src/apis/vaults.ts:184]: is Big compatible with the u128 parameter of `scaleUsingParachainGranularity`?

Comment (savudani8) [reply src/apis/vaults.ts:184]: We could remove the `private` access modifier and write a unit test

Comment (gregdhill) [reply src/apis/vaults.ts:184]: `scaleUsingParachainGranularity` takes `Big` now

Comment (gregdhill) [reply src/apis/vaults.ts:184]: yep, let's remember to do that next week :)

**Review by savudani8 — APPROVED** [review]

**Pull request merged**